### ISSUE TITLE
Add some introductory material

### ISSUE THREAD

**Issue by emdash** [stared/quantum-game#12]
I puzzled my way through 34 levels without really understanding what I was doing. The grid layout, and the straight-forward physics of mirrors constrain your choices enough that you can kindof just brute force through various permutations of things. It would be nice to have a bit more background in what this game models. Also, while the game is fun, it's not clear exactly how this relates to building quantum computers.

**Comment by sbliven** [comment]
I agree. The mouse-over explanations for each object feel more like reminders than proper explanations, and some are frustratingly vague (e.g. Faraday rotators have "different symmetries from sugar" but how?). I would have appreciated some static diagrams showing how each element modifies the phase, polarization, and entanglement of its inputs. I personally would have appreciated some math as well (not necessarily unitary transformations and linear algebra, but maybe just some basic notation for how the state changes). Maybe the introductory material could be in the form of some additional help pages rather than interrupting the game flow.

**Comment by stared** [comment]
@emdash @sbliven I agree it's needed, or at least - would be very beneficial. I even spend a day at one physics laboratory to take shots of various optical elements to show how do they look in real life.

The original idea was to give:

* description
* real-life photo
* links to further resources
* a few interactive examples, step by step showing wave vector

What blocks me is time & energy (essentially I can't work on anything alone, sadly) + there would be some initial work to rewrite help in some framework. Now it is kind of hard to edit/add.

Some parts with links are in [Quantum mechanics for high-school students](http://p.migdal.pl/2016/08/15/quantum-mechanics-for-high-school-students.html) on my blog.

**Comment by sbliven** [comment]
Reading #10 was the first time I heard about the encyclopedia. That already goes a long way to addressing my comments above. Then I noticed that clicking the mouse-over explanations also gets one to the encyclopedia. So maybe I'm just unobservant.

**Comment by stared** [comment]
@lurch Colours mean phase of a complex number, video http://nbviewer.jupyter.org/github/empet/Math/blob/master/DomainColoring.ipynb.

This whole encyclopedia needs a rewrite.

Ad Linux Format - thanks for letting me know! By any chance could you send me a photo of this page? :)
(I cannot find option to buy PDF.)
